package/Shaders/ISVolumetricLightingGenerateCS.hlsl

```diff
@@ -46,7 +110,7 @@ cbuffer PerTechnique : register(b0)
 
 	float3 depthUv = dispatchID.xyz / TextureDimensions.xyz + 0.001 * StepCoefficients[IterationIndex].xyz;
 	float depth = InverseRepartitionTex.SampleLevel(InverseRepartitionSampler, depthUv.z, 0).x;
-	float4 positionCS = float4(2 * depthUv.x - 1, 1 - 2 * depthUv.y, depth, 1);
+	float4 positionCS = float4(2 * depthUv.x - 1, 1 - 2 * depthUv.y, depth, 1);  // UVDepthToView(depthUv)
```

Comment:
Remove comment?

Reply:
That's basically the function used in other shaders. Figured we may want to consolidate in the future.